docs/versioned_docs/version-2.9.4/Examples/Template_AutomaticBookNotesFromReadwise.md

```
@@ -0,0 +1,287 @@
+		.filter((line) => line != "")
+		.map((line) => {
+			return `> ${line}`;
+		});
+
+	const attr = `\n>\\- ${sourceItem.author}, _${sourceItem.title}_`;
+
+	return `${quote}${attr}`;
+}
+
+function textFormatter(sourceText, sourceNote) {
+	let quote = sourceText
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)